lib/routes/home/widgets/payments_list/payment_item_amount.dart

```diff
@@ -97,105 +97,109 @@ void main() async {
+                    amountMsat: 4321000,
+                    status: PaymentStatus.Complete,
+                    description: "",
+                    details: PaymentDetails.ln(
+                      data: LnPaymentDetails(
+                        paymentHash: "7afeee37f0bb1578e94f2e406973118c4dcec0e0755aa873af4a9a24473c02de",
+                        label: "",
+                        destinationPubkey:
+                            "0264a67069b7cbd4ea3db0709d9f605e11643a66fe434d77eaf9bf960a323dda5d",
+                        paymentPreimage: "",
+                        keysend: false,
+                        bolt11: "",
+                      ),
                     ),
                   ),
-                ),
-                getSystemAppLocalizations(),
-              ),
+                  getSystemAppLocalizations(),
+                  800000),
             ),
 
             // Pending
             PaymentItemAmount(
               PaymentMinutiae.fromPayment(
-                const Payment(
-                  paymentType: PaymentType.Received,
-                  id: "7afeee37f0bb1578e94f2e406973118c4dcec0e0755aa873af4a9a24473c02de",
-                  feeMsat: 1234,
-                  paymentTime: 1661791810,
-                  amountMsat: 4321000,
-                  status: PaymentStatus.Pending,
-                  description: "",
-                  details: PaymentDetails.ln(
-                    data: LnPaymentDetails(
-                      paymentHash: "7afeee37f0bb1578e94f2e406973118c4dcec0e0755aa873af4a9a24473c02de",
-                      label: "",
-                      destinationPubkey: "0264a67069b7cbd4ea3db0709d9f605e11643a66fe434d77eaf9bf960a323dda5d",
-                      paymentPreimage: "",
-                      keysend: false,
-                      bolt11: "",
+                  const Payment(
+                    paymentType: PaymentType.Received,
+                    id: "7afeee37f0bb1578e94f2e406973118c4dcec0e0755aa873af4a9a24473c02de",
+                    feeMsat: 1234,
+                    paymentTime: 1661791810,
+                    amountMsat: 4321000,
+                    status: PaymentStatus.Pending,
+                    description: "",
+                    details: PaymentDetails.ln(
+                      data: LnPaymentDetails(
+                        paymentHash: "7afeee37f0bb1578e94f2e406973118c4dcec0e0755aa873af4a9a24473c02de",
+                        label: "",
+                        destinationPubkey:
+                            "0264a67069b7cbd4ea3db0709d9f605e11643a66fe434d77eaf9bf960a323dda5d",
+                        paymentPreimage: "",
+                        keysend: false,
+                        bolt11: "",
+                      ),
                     ),
                   ),
-                ),
-                getSystemAppLocalizations(),
-              ),
+                  getSystemAppLocalizations(),
+                  800000),
```

Comment:
Can you add a comma after these values and reformat code? It helps with readability & keeping the right indentation.

Reply:
Added